src/main/java/com/ldtteam/structurize/items/ModItems.java

```diff
@@ -53,5 +70,8 @@ private ModItems() { /* prevent construction */ }
         scanTool  = register("sceptersteel", () -> new ItemScanTool(ModItemGroups.STRUCTURIZE));
         tagTool   = register("sceptertag", () -> new ItemTagTool(ModItemGroups.STRUCTURIZE));
         caliper   = register("caliper", () -> new ItemCaliper(properties));
+
+        vanillaCaveAir = registerVanillaOverride(Blocks.CAVE_AIR, () -> new AirItem(Blocks.CAVE_AIR, new Item.Properties()));
+        vanillaVoidAir = registerVanillaOverride(Blocks.VOID_AIR, () -> new AirItem(Blocks.VOID_AIR, new Item.Properties()));
```

Comment:
I feel like registering them into minecraft namespace is better
But feel free to disagree

Reply:
Probably saver not to.